utils/update_tools/update_tools.py

```diff
@@ -119,6 +119,11 @@ def main():
         '--remote', help="Optional remote repository to use instead of origin"
     )
 
+    parser.add_argument(
+        'branch_name',
```

Comment:
Maybe worth to keep this consistent with the other arguments (e.g. `--branch_name <branch>`)

Reply:
In addition, I believe that, to avoid resetting master+wip by mistake, an assertion like `assert branch_name != 'master+wip'` could be useful.

Reply:
Yep, same thought goes for `master`, done